bhlib/src/main/ets/components/MainPage/BusinessPicker.ets

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BusinessModel } from './BusinessModel';
+import { BusinessDayAndTime } from './BusinessDayAndTime';
+
+@Component
+export struct BusinessPicker {
+  @Link selectedBusinessModel: BusinessModel
+  day: string [] = BusinessModel.dayArr
+  arr: number[] = [0, 1, 2, 3, 4, 5, 6]
```

Comment:
create a model class for BusinessPicker component and add all variables from this component in model class

Reply:
Done